ansible_collections/arista/avd/plugins/plugin_utils/utils/init_logging.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+import logging
+
+from ansible.utils.display import Display
+
+from .python_to_ansible_logging_handler import PythonToAnsibleHandler
+
+
+def init_pyavd_logging() -> None:
+    """
+    Specify logger parameters for pyavd and schematools.
+
+    The verbosity depends on the verbosity level passed to Ansible
+    - Ansible verbosity 0 translate to a level of warning
+    - Ansible verbosity 1 to 2 (-v to -vv) translates to a level of info
+    - Ansible verbosity 3 and above translates to a level of debug
+    """
+    pyavd_logger = logging.getLogger("pyavd")
+    schema_tools_logger = logging.getLogger("schema_tools")
```

Comment:
@ClausHolbechArista these logger will take precedence for cv_workflow in the current implementation. Let me know what you think. I will tackle logging design next to clean everything as it needs a good stratgey going forward.

Reply:
ok

Reply:
discussed on the call